doc/time-series.rst

```diff
@@ -15,6 +15,11 @@ core functionality.
     import numpy as np
     import pandas as pd
     import xarray as xr
+
+    # Use defaults so we don't get gridlines in generated docs
+    import matplotlib as mpl
+    mpl.rcdefaults()
```

Comment:
a simpler fix would be to update seaborn to v0.8, which no longer adjusts the default style